tests/qos/qos_sai_base.py

```diff
@@ -948,6 +948,12 @@ def dutConfig(
                             uplinkPortIps.append(portConfig["peer_addr"])
                             uplinkPortNames.append(intf)
 
+            if isMellanoxDevice(src_dut):
```

Comment:
Is this change only for ptf32 topo? From the code, seems it's applied to all Mellanox devices.

Reply:
Not related for ptf32 topo. Topo ptf32 do not set port speed, so the speed based on the original topology configuration. The change would filter un-matched port speed and cable length.

Reply:
Sorry I missed your reply. The PR title said ptf32 topo.
And can you please paste the test results for this change? Thanks